src/lib/utilities/find-value-if-it-includes-word.ts

```diff
@@ -0,0 +1,7 @@
+export const findValueIfItIncludesWord = (object, word) => {
+  for (var property in object) {
```

Comment:
Typically we avoid var these days. 

https://levelup.gitconnected.com/stop-using-var-to-declare-variables-in-javascript-6c0caec16f43

Outlines some basics as to the why.

This is basically just checking if an object has a key with a value in it. You can generally check that like..

```typescript
const myRadObject= {
mojo: "sodope"
};
console.log(myRadObject["mojo"] ?? "Not Dope")
> "sodope"
console.log(myRadObject["Magical"] ?? "Not Dope")
> "Not Dope"
```

Reply:
This can also probably be avoided in your code if you have types in your code.

Reply:
Also generally avoid `for( .. in ... )`. Prefer `for .. of` because it doesn't enumerate over object fields

https://stackoverflow.com/questions/500504/why-is-using-for-in-for-array-iteration-a-bad-idea

For of docs

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/for...of

Also also looking up objects this way is `O(N)` ie. it will iterate over every item in an object to find that instead of `O(1)` which will access the key directly. 

To put it another way. If you have an object with 10,000 keys you can do 10,000 checks or you can do 1 check depending on how you access the object.